tools/run_tests/xds_k8s_test_driver/framework/xds_k8s_flags.py

```diff
@@ -41,9 +41,12 @@
     "debug_use_port_forwarding",
     default=False,
     help="Development only: use kubectl port-forward to connect to test app")
+DISABLE_WORKLOAD_IDENTITY = flags.DEFINE_bool(
+    "disable_workload_identity",
+    default=False,
+    help="Disable the WorkloadIdentity feature simplify permission control")
```

Comment:
nit: To avoid double negatives, I'd recommend to inverse this flag:

```suggestion
ENABLE_WORKLOAD_IDENTITY = flags.DEFINE_bool(
    "enable_workload_identity",
    default=True,
    help="Disable the WorkloadIdentity feature simplify permission control")
```

Then it can be turned off with `--noenable_workload_identity`. `--noenable` appears to be a common pattern.